src/neural/cuda/winograd_helper.inc

```diff
@@ -48,13 +48,14 @@ __device__ __forceinline__ float activate(float cVal, ActivationFunction activat
     case SIGMOID:
       cVal = 1.0f / (1.0f + exp(-cVal));
       break;
-    case SELU:
+    case SELU: {
```

Comment:
You saw this warning as well I see :)

Reply:
The last commit completely cleans the build with gcc 9.4